docs/source/en/model_doc/table-transformer.mdx

```diff
@@ -43,6 +42,12 @@ alt="drawing" width="600"/>
 This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be
 found [here](https://github.com/microsoft/table-transformer).
 
+## Resources
+
+<PipelineTag pipeline="object-detection"/>
+
+- A demo notebook for the Table Transformer can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/Table%20Transformer).
+- It turns out padding of images is quite important for detection. An interesting Github thread with replies from the authors can be found [here](https://github.com/microsoft/table-transformer/issues/68). 
```

Comment:
I wonder if it'd be easier for users if we summarized the most important facts or points from the GitHub thread in a separate Tips section instead of sending the user off to read/scan the discussion. What do you think @sgugger ?

Reply:
I'm fine with both.

Reply:
Ah ok! I'm slightly in favor of summarizing the thread, but I don't really have a strong preference, so I'll leave it up to you :)